josue/js/app.js

```diff
@@ -0,0 +1,52 @@
+import FormValidator from './Formvalidator.js';
+
+const form = document.querySelector('.form');
+const signInBtn = document.querySelector("#signBtn");
+const registerBtn = document.querySelector("#registerBtn");
+const submitBtn = document.querySelector("#submit");
+const registerFields = document.querySelector("#register");
+let fields = ['name','phone'];
+
+registerFields.remove();
+signInBtn.addEventListener("click",e=>{
+    registerFields.classList.add("register--hidden");
+    registerFields.remove();
+    submitBtn.innerText = "Sign In"
+    fields = ['name','phone']
```

Comment:
you already assigned this array to the variable `fields` on line 8, why don't you make it a constant and delete this line? On line 16 you will still have a reference to the right array

Reply:
Right, I'll change it to a const. The reason why is reasigned is because the user can change from the register form and then getting back to login, if we don't refresh the fields the event listeners of the register form will be active.